CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased changes
 
+## v0.2.0 - 2024-05-13
+
```

Comment:
(not blocking) would it be useful, going forward, to use the pattern in https://keepachangelog.com/en/1.1.0/ so we clearly communicate breaking changes vs. additions or other things?

Reply:
Yes, that seems sensible.

I'll add headings to this release now, and we can change the formatting in another PR.